builders/integrations/bridges/wormhole.md

```diff
@@ -0,0 +1,422 @@
+
+If you recall the list of components, the third is the Redis database component. Everything that has to do with the database is hidden from the user as well, since the _relayer-engine_ package will write & read from it, then inject any relevant data back into the plugin code. There’s nothing more that needs to be done for it, so feel free to take a break.
+
+
+### Setting up the Executor Component {:setting-up-the-executor-component}
+
+Finally, you must handle the executor component. Recall that the executor component takes workflow data from the Redis database and does some sort of execution action with that data. For most relayers, this execution will involve an on-chain transaction, since a relayer acts as a trustless oracle for VAAs.
+
+The _relayer-engine_ package helps handle the wallets for the plugin. Currently, the package only supports Solana and EVM wallets, but with further development more chains will be supported. But it’s not impossible to integrate NEAR or Algorand into the relayer, since you would just have to write your own wallet handling system in addition to the one already provided by the package.
+
+To work with the built-in wallet handling system provided by the package, open the file at _example-project/relayer-engine-config/executor.json.example_. This example script is provided to show you how to format your private keys (the current key is provided by Wormhole). 
+
+Rename the example file to _executor.json_. In the _privateKeys_ object of _executor.json_, replace the content of each array with your private key. The account of the private key entries will be the one that pays for execution fees in the relayer’s executor component.
+
+Please manage your keys with care, as exposing them can result in loss of funds. While _executor.json_ is ignored by git in this repository, please be sure that the wallet you are using for testnet has no mainnet funds.
+
+
+```javascript
+{
+   "privateKeys": {
+       "16": [
+           "0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
+       ],
+       "2": [
+           "0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
+       ],
+       "5": [
+           "0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
+       ],
+       "6": [
+           "0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
+       ],
+       "10": [
+           "0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
+       ],
+
+   }
+}
+```
+
+
+If you are using a chain that wasn’t listed in the EVM TestNet table above, you will have to add your own array. The key for this array should be the Wormhole _chainId_ of the other EVM that you chose to deploy on before. For example, if you deployed on the Fantom TestNet, you would add the following object, since the Wormhole _chainId_ of the Fantom TestNet is 10. 
```

Comment:
it doesn't look `chainId` is used as a particular variable name, but if I am wrong please ignore this comment... I think it should be chain ID (as it is in the table)